File: python/CWE-798.py
import requests

def make_request():
    # Жестко заданные учетные данные (уязвимость CWE-798)
    username = "admin"
    password = "password123"

    # URL для отправки запроса
    url = "https://example.com/api/data"

    # Выполнение запроса с использованием жестко заданных учетных данных
    response = requests.get(url, auth=(username, password))

    if response.status_code == 200:
        print("Request successful!")
        print("Data:", response.json())
    else:
        print("Request failed with status code:", response.status_code)

if __name__ == "__main__":
    make_request()

File: PHP/CWE-798.php
<?php
// Hard-coded credentials in PHP
$username = "admin";
$password = "password123";

function authenticate($user, $pass) {
    global $username, $password;
    if ($user === $username && $pass === $password) {
        echo "Authentication successful!";
    } else {
        echo "Authentication failed.";
    }
}

// Example usage
authenticate("admin", "password123");
?>
